packages/cysync-core/src/pages/OnBoarding/OnboardingPageLayout.tsx

```diff
@@ -78,6 +80,9 @@ export const OnboardingPageLayout: React.FC<OnboardingPageLayoutProps> = ({
       text={text}
       currentState={currentState}
       totalState={totalState}
+      title={title}
+      subTitle={subTitle}
+      version={`ver ${(window as any).cysyncEnv.VERSION}`}
```

Comment:
Define `cysyncEnv` in a `types.d.ts` file, so you won't need to typecast it as any. Refer `types.d.ts` in `desktop-ui` package.